build.sh

```diff
@@ -0,0 +1,7 @@
+# build instrumentation tool
+go build -a -o otel
+TOOL=$(pwd)/otel
+# compile-time instrumentation via toolexec
+cd demo
+go build -a -toolexec=$TOOL -o demo .
```

Comment:
What do you think about adding `work` switch to output compiler working directory location? Then, we could mention that in the Readme
```suggestion
go build -a -work -toolexec=$TOOL -o demo .
```